astroplan/skycalc.py

```diff
@@ -0,0 +1,179 @@
+            Note: This radiance component represents an instrumental effect.
+            The emission is provided relative to the other model components.
+            To obtain the correct absolute flux, an instrumental response curve
+            must be applied to the resulting model spectrum.
+            See section 6.2.4 in the SkyCalc documentation at
+            http://localhost/observing/etc/doc/skycalc/
+            The_Cerro_Paranal_Advanced_Sky_Model.pdf
+        therm_t1, therm_t2, therm_t3 : float. default = 0.0
+            Temperature in K
+        therm_e1, therm_e2, therm_e3 : float in range [0.0, 1.0]. default = 0.0
+        vacair : {'vac', 'air'}. default = 'vac'
+            In regards to the wavelength grid.
+        wmin : float in range [300.0, 30000.0]. default = 300.0
+            Minimum wavelength (nm) in the wavelength grid.
+            Must be < wmax.
+        wmax : float in range [300.0, 30000.0]. default = 2000.0
+            Maximum wavelength (nm) in the wavelength grid.
+            Must be > wmin.
+        wgrid_mode : {'fixed_spectral_resolution', 'fixed_wavelength_step', 'user'}.
+            default = 'fixed_wavelength_step'
+            Mode of the wavelength grid.
+        wdelta : float in range [0, 30000.0]. default = 0.1
+            Wavelength sampling step dlam (i.e. nm/step)
+        wres : int in range [0,1.0e6]. default = 20000
+            lam/dlam where dlam is wavelength step.
+        wgrid_user : list of floats
+            default = [500.0, 510.0, 520.0, 530.0, 540.0, 550.0]
+        lsf_type : {'none', 'Gaussian', 'Boxcar'}. default = 'none'
+            Line spread function type for convolution.
+        lsf_gauss_fwhm : float. default = 5.0
+            Gaussian full-width half-max for line spread function wavelength bins.
+        lsf_boxcar_fwhm : float. default = 5.0
+            Boxcar full-width half-max for line spread function wavelength bins.
+        observatory : {'paranal', 'lasilla', 'armazones'}. default = 'paranal'
+            Observatory where observation takes place.
+
+        Returns
+        -------
+        trans_waves, transmission: tuple of arrays of floats
+            'trans_waves' is an array of wavelengths in angstroms (float),
+            'transmission' is an array of fractional atmospheric
+            transmittance (float).
+
+        """
+        params = locals()
+
+        if params['observatory'] == 'lasilla':
+            params['observatory'] = '2400'
+        elif params['observatory'] == 'paranal':
+            params['observatory'] = '2640'
+        elif (params['observatory'] == '3060m' or
+              params['observatory'] == 'armazones'):
+            params['observatory'] = '3060'
+        else:
+            raise ValueError('Wrong Observatory name, please refer to the '
+                             'skycalc_cli documentation.')
+
+        # import packages not required by astroplan
+        import os
```

Comment:
`os` and `json` are both built-in modules in Python and can be moved to the top of this module.